apps/vr-tests-react-components/src/stories/Progress.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Progress } from '@fluentui/react-progress';
+import { TestWrapperDecoratorNoAnimation } from '../utilities/TestWrapperDecorator';
+
+storiesOf('Progress converged', module)
+  .addDecorator(TestWrapperDecoratorNoAnimation)
+  .addStory('Indeterminate', () => <Progress className="test-class" />, {
+    includeDarkMode: true,
+    includeHighContrast: true,
+    includeRtl: true,
+  })
+  .addStory('Indeterminate with thickness medium', () => <Progress className="test-class" thickness="medium" />, {
+    includeDarkMode: true,
+    includeHighContrast: true,
+    includeRtl: true,
+  })
+  .addStory('Indeterminate with thickness large', () => <Progress className="test-class" thickness="large" />, {
+    includeDarkMode: true,
+    includeHighContrast: true,
+    includeRtl: true,
+  })
```

Comment:
You probably don't need all of these variants for the size tests. They're already covered by the basic test.
```suggestion
  .addStory('Indeterminate with thickness large', () => <Progress className="test-class" thickness="large" />)
```